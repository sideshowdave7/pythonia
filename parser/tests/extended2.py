def a():
 for i in range(10):
  yield i, i*i

for i in a():
 print(i)

