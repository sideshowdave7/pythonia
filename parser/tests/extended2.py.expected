(program
 (def
  (a)
  (suite
   (for i (indexed range (called 10)) (suite (yield i (term i ("*" i)))))))
 (for i (indexed a (called)) (suite (expr (indexed print (called i))))))

